qualtran/simulation/xcheck_classical_quimb_test.py

```diff
@@ -0,0 +1,49 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import qualtran.testing as qlt_testing
+from qualtran.bloqs.arithmetic import Add
+from qualtran.bloqs.basic_gates import CNOT, Toffoli, XGate
+from qualtran.simulation.xcheck_classical_quimb import flank_with_classical_vectors
+
+
+def test_xgate():
+    x = XGate()
+    x_tt = flank_with_classical_vectors(x, {'q': 0})
+    assert x_tt.tensor_contract() == 1.0
+
+    result = flank_with_classical_vectors(x, in_vals={'q': 0}, out_vals={'q': 0}).tensor_contract()
+    assert result == 0.0
+
+
+def test_cnot():
```

Comment:
Is it worth parametrizing these tests over inputs / outputs.

Reply:
these tests are in the test file associated with the flank function rather than for e.g. cnot. I don't think it would give us enough added coverage to try all the values vs the extra runtime.